### PR TITLE
Feature/check in out

### PR DESCRIPTION
**Pull request by mateomotriz**
This is taking me too much time, and its pretty simple:

the class getTextBox is working fine if I remove the TextField. But the thing is that I dont know whats wrong with the TextField, it just doesnt want to appear on the layout. 

Once solved that, I will solve small problems like paddings, design, and small arrangements.

### CODE REVIEW

File: lust/lib/widgets/getTextBox.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: <Widget>[
+        Text(
+          _title,
+          style: TextStyle(fontSize: 20),
+        ),
+        TextField(
+          decoration: InputDecoration(
+            //labelText: "whatever",
+            labelStyle: TextStyle(fontSize: 16, color: Colors.black45),
```
Comment (worldpotato) [inline lust/lib/widgets/getTextBox.dart]: try only black not black45

File: lust/lib/widgets/getTextBox.dart [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            hintText: _hint,
+            hintStyle: TextStyle(fontSize: 14, color: Colors.black12),
```
Comment (worldpotato) [inline lust/lib/widgets/getTextBox.dart]: the same. Try it with Colors.black

File: lust/lib/widgets/getTextBox.dart [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  //TextBox(this._text);
+  //final String _text;
+  //String inputUser;
+
+  static Widget getTextBox(BuildContext context, String _title, String _hint) {
+    String inputUser;
+
+    return new Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: <Widget>[
```
Comment (worldpotato) [inline lust/lib/widgets/getTextBox.dart]: mhm. Maybe try:
Dont use `children: <Widget>[`
use only `children: [`

Thats how I did it in the Capacity view

File: lust/lib/widgets/getTextBox.dart [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        Text(
+          _title,
+          style: TextStyle(fontSize: 20),
+        ),
+        TextField(
```
Comment (andreBrauni) [inline lust/lib/widgets/getTextBox.dart]: wrap an expanded around the textfield
like so: https://stackoverflow.com/questions/54269313/fill-row-with-textfield

**Comment by mateomotriz** [comment]
CI not working because when the keyboard is shown, there is some kind of pixels overflow.

**Pull request closed without merging**